docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,59 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged
+   into `main`.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
```

Comment:
Currently on develop we are using `meta-zarhus-distro/conf/distro/include/zarhus-distro-common.conf`.
Does this document apply to `meta-zarhus` or to projects that are based on `meta-zarhus`?

Reply:
Generally this document applies to `meta-zarhus`, but we should specify the actual path in the repository's documentation